src/main/pages/che-6/workspace-admin/secure-servers.adoc

```diff
@@ -1,48 +0,0 @@
----
-title: "Secure Servers"
-keywords: workspace, runtime, recipe, kubernetes, openshift, stack, servers, server, secure server
-tags: [workspace, runtime, docker, kubernetes]
-sidebar: che_6_docs
-permalink: che-6/secure-servers.html
-redirect_from: secure-servers.html
-folder: che-6/workspace-admin
----
-
-Now that you have made yourself familiar with link:servers.html#secure-server[secure servers] concept, let’s take a closer look at enabling this functionality and implementation details.
```

Comment:
It contains a lot of useful and actual for Che 7 documentations, are they all reviewed and migrated to che 7 docs?

Reply:
It needs to be done. We (as the documentation team) don't know what is still relevant.